test/server/logarchiver/testlogarchiver.cpp

```diff
@@ -91,15 +92,16 @@ void TestLogArchiver::testCopyLogsTo(void) {
         bool tooDeep = false;
         IoHelper::getDirectorySize(tempDir.path, tempDirSize, err, 0);
         CPPUNIT_ASSERT(err == IoErrorSuccess || err == IoErrorMaxDepthExceeded);
-        CPPUNIT_ASSERT_EQUAL(logDirsize, tempDirSize);
+        CPPUNIT_ASSERT_GREATER(logDirsize, tempDirSize);
     }
 
     {  // Test without archivedLogs
-        TemporaryDirectory tempDir;
+        TemporaryDirectory tempDir("logArchiver");
         SyncPath logDir = Log::instance()->getLogFilePath().parent_path();
 
         // create a fake log file
         std::ofstream logFile(tempDir.path / "test.log");
+        CPPUNIT_ASSERT(logFile.is_open());
         for (int i = 0; i < 10; i++) {
```

Comment:
General remark: I would keep for-loop counters as low as possible everywhere it's applicable.

Reply:
[b2d2412](https://github.com/Infomaniak/desktop-kDrive/pull/139/commits/b2d241289315e2063e52cf1f4d5bcf33f023fea8)